htlcswitch/link.go

```diff
@@ -2060,6 +2060,10 @@ func (l *channelLink) updateCommitTx() error {
 		return err
 	}
 
+	if err := l.ackDownStreamPackets(); err != nil {
```

Comment:
Same comment here re having the commit body populated with the fragment related to this commit from the main PR body.